### PR TITLE
sys-process/parallel: bump to 20200422

### PR DESCRIPTION
**Pull request by thesamesam**
I will be working on enabling some tests in future, see bug https://bugs.gentoo.org/720878, but this is not a quick job as explained in that bug.

### CODE REVIEW

File: sys-process/parallel/parallel-20200422.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+IUSE=""
+
+RDEPEND="dev-lang/perl:=
+	dev-perl/Devel-Size
+	virtual/perl-Data-Dumper
+	virtual/perl-File-Temp
+	virtual/perl-IO"
+DEPEND="${RDEPEND}"
+
+DOCS="NEWS README"
```
Comment (thesamesam) [inline sys-process/parallel/parallel-20200422.ebuild:22]: I'm going to check if this is actually needed.

File: sys-process/parallel/parallel-20200422.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+EAPI=7
+
+DESCRIPTION="A shell tool for executing jobs in parallel locally or on remote machines"
+HOMEPAGE="https://www.gnu.org/software/parallel/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
```
Comment (thesamesam) [inline sys-process/parallel/parallel-20200422.ebuild:13]: Dropping this in a minute.

File: sys-process/parallel/parallel-20200422.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+src_install() {
+	default
+
+	# See src/Makefile.am for this one:
+	rm -f "${ED}"/usr/bin/sem || die
+	dosym ${PN} /usr/bin/sem
+}
+
+pkg_postinst() {
```
Comment (thesamesam) [inline sys-process/parallel/parallel-20200422.ebuild:36]: [optfeature](https://devmanual.gentoo.org/eclass-reference/eutils.eclass/)?

File: sys-process/parallel/parallel-20200422.ebuild [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+src_configure() {
+	econf --docdir="${EPREFIX}"/usr/share/doc/${PF}/html
+}
+
+src_install() {
+	default
+
+	# See src/Makefile.am for this one:
+	rm -f "${ED}"/usr/bin/sem || die
```
Comment (juippis) [inline sys-process/parallel/parallel-20200422.ebuild:29]: We dislike using `-f` when the action can be predicted. You should know when this file exists, and how. 

So just remove the `-f` option here. Rest looks good.

Comment (thesamesam) [reply sys-process/parallel/parallel-20200422.ebuild]: Gone.

**Pull request closed without merging**